src/Uno.UI.RemoteControl/RemoteControlClient.cs

```diff
@@ -80,6 +81,11 @@ private void RegisterProcessor(IRemoteControlProcessor processor)
 			_processors[processor.Scope] = processor;
 		}
 
+		public void RegisterPreProcessor(IRemoteControlPreProcessor preprocessor)
```

Comment:
Do we expect this to be called in a multi-threaded manner?

Reply:
Hardly expect this to be called at all. I can only think of (have) one, niche scenario where this is necessary.